app/models/concerns/seasonal.rb

```diff
@@ -26,6 +26,17 @@ module Seasonal
         set_current_year
       end
     end
+
+    rails_admin do
+      list do
+        field :year
+        field :student do
+          queryable true
+          searchable [:first_name, :last_name]
+        end
+        include_all_fields
```

Comment:
I don't really understand what this does, can you explain it?

Reply:
Oh nvm I got it. It makes it so all default fields are included, even after you explicitly include year and student. 👍